samples/nrf9160/modem_shell/overlay-ppp.conf

```diff
@@ -95,6 +95,7 @@ CONFIG_NET_L2_PPP_MGMT=y
 CONFIG_NET_L2_PPP_OPTION_MRU=y
 CONFIG_NET_L2_PPP_OPTION_SERVE_IP=y
 CONFIG_NET_L2_PPP_OPTION_SERVE_DNS=y
+CONFIG_NET_L2_PPP_MAX_TERMINATE_REQ_RETRANSMITS=1
```

Comment:
Should be noted, to get this config working, following fix is needed from Zephyr: https://github.com/zephyrproject-rtos/zephyr/pull/44592